app/models/taxonomy.rb

```diff
@@ -210,6 +210,10 @@ def params_objects
     (self.send("#{type.downcase}_parameters".to_sym).authorized(:view_params) + taxonomy_inherited_params_objects.to_a.reverse!).uniq {|param| param.name}
   end
 
+  def notification_recipients_ids
+    self.flat_map(&:subtree).flat_map(&:users).uniq
```

Comment:
@timogoebel I suppose this was supposed to be `subtree.flat_map(&:users).uniq`? Tests are failing because `self.flat_map` won't work as `self` is not a collection.